### DISCUSSION TITLE
Kratos v1.2.0: "State" property of a Create Flow response

### DISCUSSION THREAD

**Discussion by edeal-at-au** [ory/kratos#3973] [Q&A]
It appears that "State" is a required property of a Create Flow response (Create Registration/Login/Recovery Flows), however, it seems only Create Recovery Flow populating the "State" property and, as a result, kratos-client-go returning an error for the following calls: 
CreateNativeRegistrationFlow, CreateBrowserRegistrationFlow, CreateNativeLoginFlow, CreateBrowserLoginFlow
Here is the error: "no value given for required property state"
Can someone shed a light on this, please?

**Comment by heartical** [comment]
same! how to fix this??
